go.mod

```diff
@@ -19,7 +19,7 @@ retract (
 replace (
 	github.com/cihub/seelog => github.com/cihub/seelog v0.0.0-20151216151435-d2c6e5aa9fbf // v2.6
 	github.com/coreos/go-systemd => github.com/coreos/go-systemd v0.0.0-20180202092358-40e2722dffea
-	github.com/docker/distribution => github.com/docker/distribution v2.7.1-0.20190104202606-0ac367fd6bee+incompatible
+	github.com/docker/distribution => github.com/distribution/distribution v2.8.0+incompatible
```

Comment:
Instead of using a replace, why not use the dependency directly? See ["Why we should avoid replace directives"](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/gomodreplace.md#why-we-should-avoid-replace-directives)

Reply:
To be honest I'm not sure of the reasoning behind the decision of using replace here. I'm making this PR because we investigated that distribution/distribution was using a lot of cpu/memory at init time and made a PR to improve this

Reply:
I see. It looks like this was introduced in #5292, with no specific reasoning as to why it was added. I think it's worth it to try to remove it, and see if it builds (I have done that successfully with other `replace`s added by that PR).